eclair-core/src/main/scala/fr/acinq/eclair/payment/send/OfferPayment.scala

```diff
@@ -138,6 +138,7 @@ private class OfferPayment(replyTo: ActorRef,
             Behaviors.stopped
           case None =>
             context.spawnAnonymous(CompactBlindedPathsResolver(router)) ! Resolve(context.messageAdapter[Seq[ResolvedPath]](WrappedResolvedPaths), payload.invoice.blindedPaths)
+            context.log.info(s"BOLT12 invoice: ${payload.invoice}")
```

Comment:
nit: remove?

Reply:
Not really. It's really useful for debugging. How else could we know what kind of invoices our counterparties send to us?

Reply:
I don't understand...? Why is it useful to log the Bolt 12 invoice we receive? If the payment completes it will be stored in our payments DB. If the payment fails, do you really need the invoice to investigate why it failed? It will generally be easy to figure out what fails without it?